test/E2ETests/E2ETests/Helpers/TableHelpers.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Data.Tables;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Functions.Tests.E2ETests;
+
+namespace Microsoft.Azure.Functions.Worker.E2ETests.Helpers
+{
+    public static class TableHelpers
+    {
+        private static readonly TableClient _tableClient;
+        static TableHelpers()
+        {
+            var builder = new System.Data.Common.DbConnectionStringBuilder
+            {
+                ConnectionString = Constants.Tables.TablesConnectionStringSetting
+            };
+            var tableName = Constants.Tables.TableName;
+            _tableClient = new TableClient(Constants.Tables.TablesConnectionStringSetting, tableName);
+
+        }
+
+        public async static Task CreateTable()
+        {
+            _ = await _tableClient.CreateIfNotExistsAsync();
+        }
+
+        public async static Task DeleteTable()
+        {
+            _ = await _tableClient.DeleteAsync();
```

Comment:
```suggestion
            await _tableClient.DeleteAsync();
```

don't need to assign if not used